src/main/java/task/Event.java

```diff
@@ -0,0 +1,20 @@
+package task;
+
+import java.time.LocalDate;
+public class Event extends Task {
+    private LocalDate start;
+    private LocalDate end;
+
+    public Event(String instruction, String start, String end) {
+        super(instruction.substring(6));
```

Comment:
I think this line can be hard to understand, especially as the reader may not know that all Event instructions start with the word 'event' (and similarly for the other Task subclasses). Also, might there be issues if the instruction is just "event", and there is no 6th index? Perhaps using instruction.split(" ", 2) might be better to separate out the words.